mmf/utils/features/visualizing_image.py

```diff
@@ -0,0 +1,515 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Is there a reason this can't also go into utils/visualize.py?

Reply:
I could definitely put it there, I just did not want to overcrowd that file with feature-displaying-specific code, do you think I should move it?